pkg/nsx/services/common/types.go

```diff
@@ -46,6 +46,7 @@ const (
 	LabelDefaultSubnetSet           string = "nsxoperator.vmware.com/default-subnetset-for"
 	LabelDefaultVMSubnet            string = "VirtualMachine"
 	LabelDefaultPodSubnetSet        string = "Pod"
+	TagSubnetFor                    string = "nsx-op/subnet-for"
```

Comment:
How about `nsx-op/created_for`

Reply:
subnet_created_for is better?

Reply:
In future, we may have other cases to use created_for, similar with NCP